### PR TITLE
Add wallet sync info into Overview page

### LINKED ISSUE #376: add sync card to the overview page and enable auto sync

### PR DESCRIPTION
**Pull request by JustinBeBoy**
Resolve #376 

**Screenshot**

image

### CODE REVIEW

**Comment by dreacot** [comment]
the bottom controls are not well positioned

Screenshot 2024-01-16 at 6 20 14 AM

**Comment by dreacot** [comment]
when a new wallet is create also enable auto sync

**Comment by dreacot** [comment]
When the backup now button is visible, it affects the positioning of the controls

Screenshot 2024-01-17 at 5 08 57 AM

clicking on the controls open the wallet info page rather than actually doing the intended job of the controls which is switching the slider
Screenshot 2024-01-17 at 5 09 47 AM

File: ui/page/root/overview_page.go [hunk 1]
```diff
@@ -255,6 +270,28 @@ func (pg *OverviewPage) HandleUserInteractions() {
...
 	}
+
+	if pg.infoSyncWalletsSlider.Clicked() {
+		curSliderIndex := pg.infoSyncWalletsSlider.GetSelectedIndex()
+		infoSyncWallet := pg.listInfoWallets[curSliderIndex]
+
+		// pg.showNavigationFunc(true)
+		// walletCallbackFunc := func() {
+		// 	pg.showNavigationFunc(false)
+		// }
```
Comment (dreacot) [inline ui/page/root/overview_page.go:281]: remove commented code

File: ui/page/root/overview_page.go [hunk 2]
```diff
@@ -255,6 +270,28 @@ func (pg *OverviewPage) HandleUserInteractions() {
...
+		callback := func() {
+			pg.showNavigationFunc(false)
+			pg.ParentNavigator().CloseCurrentPage()
+		}
+		swmp := wallet.NewSingleWalletMasterPage(pg.Load, infoSyncWallet.GetWallet(), callback)
+		pg.ParentNavigator().Display(swmp)
+		// swmp.Display(privacy.NewAccountMixerPage(pg.Load, selectedWallet)) // Display mixer page on the main page.
+		// swmp.PageNavigationTab.SetSelectedSegment(values.String(values.StrStakeShuffle))
+
+		fmt.Println("----------------------------->")
```
Comment (dreacot) [inline ui/page/root/overview_page.go:293]: remove commented code and debug code

**Review by dreacot — CHANGES_REQUESTED** [review]

**Comment by JustinBeBoy** [comment]
> 1. When the backup now button is visible, it affects the positioning of the controls
> 2. Clicking backup now from the sync card doesn't show the correct page after trying to leave the wallet info page, clicking the back button on the wallet info page shows the backup checklist
> 
> Screenshot 2024-01-17 at 5 08 57 AM
> clicking on the controls open the wallet info page rather than actually doing the intended job of the controls which is switching the slider Screenshot 2024-01-17 at 5 09 47 AM

done

**Review by dreacot — APPROVED** [review]

**Pull request merged**